pkg/operator/ceph/cluster/controller.go

```diff
@@ -341,17 +344,46 @@ func (c *ClusterController) onK8sNodeUpdate(oldObj, newObj interface{}) {
 		logger.Warningf("Expected Node but handler received %#v", oldObj)
 		return
 	}
+	newNodeSchedulable := k8sutil.GetNodeSchedulable(*newNode)
+	oldNodeSchedulable := k8sutil.GetNodeSchedulable(*oldNode)
+	clusters := c.getTenantClusters(newNode)
 
-	if k8sutil.GetNodeSchedulable(*newNode) == false {
-		logger.Debugf("Skipping cluster update. Updated node %s is unschedulable", newNode.Labels[apis.LabelHostname])
-		return
+	for _, cluster := range clusters {
+		logger.Debugf("NodeUpdate: oldNodeSchedulable: %v, newNodeSchedulable: %v", oldNodeSchedulable, newNodeSchedulable)
+		if cluster.Info.IsInitialized() {
```

Comment:
This method is getting log. Could you split out the check for nodes in maintenance to another method?

Reply:
+1